web/src/Form.js

```diff
@@ -1,39 +1,43 @@
 import React, { Component } from "react";
 import TextboxButton from "./TextboxButton";
 import Spinner from "./Spinner";
-import "./Form.css";
+import glamorous from "glamorous";
 
-class Form extends Component {
-  get inputStyle() {
-    return this.props.loading ? { display: "none" } : {};
-  }
-
-  get spinnerStyle() {
-    return this.props.loading
-      ? { opacity: this.props.alpha }
-      : { display: "none" };
-  }
-
-  get formStyle() {
-    return {
-      height: `${this.props.height}px`,
-      width: `${this.props.width}px`,
-      borderRadius: `${this.props.radius}px`,
-      backgroundColor: `rgba(10,172,142,${this.props.alpha})`
-    };
-  }
+const FormContainer = glamorous.div(
+  {
+    border: "1px solid rgba(10, 172, 142, 1)",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    overflow: "hidden",
+    backgroundColor: "white",
+    marginTop: "15px"
+  },
+  props => ({
+    height: `${props.height}px`,
+    width: `${props.width}px`,
+    borderRadius: `${props.radius}px`,
+    backgroundColor: `rgba(10,172,142,${props.alpha})`
+  })
+);
 
+class Form extends Component {
   render() {
     return (
-      <div className="form" style={this.formStyle}>
-        <Spinner style={this.spinnerStyle} />
+      <FormContainer
+        height={this.props.height}
+        width={this.props.width}
+        radius={this.props.radius}
+        alpha={this.props.alpha}
+      >
```

Comment:
I would add `const {height, width, radius, alpha, loading} = this.props;` to avoid using `this.props` every time.